cmd/tk/main.go

```diff
@@ -77,18 +77,6 @@ func setupConfiguration(baseDir string) *v1alpha1.Environment {
 			if verbose {
 				fmt.Print(err)
 			}
-		// no spec.json is found, try parsing main.jsonnet
```

Comment:
This is function is now only used in `cmd/tk/env.go:envSetCmd()`, which has no affinity with inline environments.